fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,154 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .sites import BaseSiteContentModel, BaseControlledSiteContentModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseSiteContentModel):
+    """Model for notes associated to each dictionary entry."""
+
+    # from fv:notes,fv:general_note, fv:cultural_note, fv:literal_translation, fv-word:notes, fv-phrase:notes
+    text = models.TextField()
```

Comment:
Sorry if this was already discussed, but do we need large text notes? If there is a max length currently enforced in the ui, maybe we could stick to that

Reply:
Max length can be added after research on basis of ticket FW-4196. Marking it resolved for now.